client/coral-framework/actions/embedStream.js

```diff
@@ -0,0 +1,22 @@
+import {base, handleResp, getInit} from '../helpers/response';
```

Comment:
could we get a comment block at the top here explaining generally what this file does? `embedStream` seems a bit vague (and I'm a bit dense)